src/cart/src/cart/crt_rpc.h

```diff
@@ -288,10 +293,20 @@ struct crt_rpc_priv {
 
 /* CRT internal opcode definitions, must be 0xFF00xxxx.*/
 enum {
-	__FIRST  = CRT_PROTO_OPC(CRT_OPC_INTERNAL_BASE, 0, 0) - 1,
+	__FIRST_INTERNAL  = CRT_PROTO_OPC(CRT_OPC_INTERNAL_BASE, \
+					CRT_PROTO_INTERNAL_VERSION, 0) - 1,
 	CRT_INTERNAL_RPCS_LIST,
 };
 
+#define CRT_OPC_FI_BASE		0xF1000000UL
+
+/* CRT internal opcode definitions, must be 0xFF00xxxx.*/
+enum {
+	__FIRST_FI  = CRT_PROTO_OPC(CRT_OPC_FI_BASE,		\
```

Comment:
this one seems correct to me.   Why do you have the \ in this line?   This isn't a #define